test/functional/wallet_importmulti.py

```diff
@@ -101,16 +103,16 @@ def run_test (self):
         assert_equal(address_assert['ismine'], False)
         assert_equal('timestamp' in address_assert, False)
 
-
-        # Address + Public key + !Internal
+        # Address + Public key + !Internal(explicit)
```

Comment:
Don't delete this line
```suggestion

        # Address + Public key + !Internal(explicit)
```